### PR TITLE
update sponsors, modify view

### PR DESCRIPTION
**Pull request by hzoo**
Could be better but the current site takes too much space anyway? Should do a separate donator page as well (feel free to modify though)

image

### CODE REVIEW

File: website/pages/en/index.js [hunk 1]
```diff
@@ -297,6 +269,20 @@ const Hero = ({ language }) => (
       </div>
 
       <MiniRepl language={language} />
+
+      <h3>Special Sponsor</h3>
+      <div class="sponsors-tier">
+        <a href="https://www.handshake.org" title="Handshake" target="_blank">
+          <img
+            src="https://handshake.org/images/landing/logo-dark.svg"
```
Comment (nicolo-ribaudo) [inline website/pages/en/index.js]: I slightly prefer `https://handshake.org/images/landing/logo-light.svg`

File: website/pages/en/index.js [hunk 2]
```diff
@@ -230,29 +203,28 @@ const OpenCollectiveSponsors = ({ language }) => {
...
             title="Gold Sponsors (Open Collective)"
             tier="gold-sponsors"
-            button={ocButton}
           />
           <SponsorTier
-            type="other"
-            title="Misc Sponsors"
-            tier="other-sponsors"
+            type="opencollective"
+            title="Silver Sponsors (Open Collective)"
```
Comment (nicolo-ribaudo) [inline website/pages/en/index.js]: Is this title shown anywhere? If not, we could maybe do something like this: (it's in the "title" of the link, that is shown on hover)

schermata da 2019-01-08 23-40-56

File: website/pages/en/index.js (hunk 1 above)
Comment (hzoo) [reply website/pages/en/index.js]: didn't think of that!

**Comment by hzoo** [comment]
Merging for now, incremental steps!

**Pull request merged**